Documentation/operations/performance/tuning.rst

```diff
@@ -699,6 +699,29 @@ distributions offer kernel images with this option set or you can re-compile
 the Linux kernel. ``CONFIG_PREEMPT_NONE=y`` is the recommended setting for
 server workloads.
 
+Scaling
```

Comment:
I'm not sure `Scaling` is a good title here, especially that there is a separate Scalability section next to the Tuning Guide. I think something like:
```
Kubernetes
==========

Set scheduling mode
-------------------
```
would be better.

Reply:
Sounds good, thanks.